apps/api/src/app/controllers/__tests__/__snapshots__/userManagementQueries.e2e.spec.ts.snap

```diff
@@ -1,5 +1,166 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`User Management Queries when fetching schemas (/admin) should return the expected result 1`] = `
+Array [
+  Object {
+    "description": "A tag is used to group related resources or notes in a non-hierarchical manner",
+    "label": "Tag",
+    "link": "api/tags",
+    "schema": Object {
+      "id": Object {
+        "coscradDataType": "NON_EMPTY_STRING",
+        "isArray": false,
+        "isOptional": false,
+      },
+      "label": Object {
+        "coscradDataType": "NON_EMPTY_STRING",
+        "isArray": false,
+        "isOptional": false,
+      },
+    },
+    "type": "tag",
+  },
+  Object {
+    "description": "A category tree provides a hierarchical organization of resources",
+    "label": "Category",
+    "link": "api/treeOfKnowledge",
+    "schema": Object {
+      "children": Object {
+        "coscradDataType": Object {
```

Comment:
Beware- we need to mark this as recursive somehow. We can "hard-wire" this on the front-end as an exceptional case, so let's not get carried away.